backend/api.test/Mocks/StidServiceMock.cs

```diff
@@ -16,5 +21,62 @@ public async Task<Position> GetTagPosition(string tag, string installationCode)
                 z: 0
             );
         }
+
+        public virtual async Task<Area> GetTagArea(string tag, string installationCode)
+        {
+            await Task.CompletedTask;
```

Comment:
This might fail in the future if we do more validation on the areas that are valid within a tag for a given mission/robot, but it is an OK fix for now.